File: clear_images.py
#! /usr/bin/env python3

from os import listdir, path, remove

IMG_FOLDER = "imgs"

print("Clearing all saved images")
if input("Are you sure? (y/N)") != "y":
    exit()

[remove(path.join(IMG_FOLDER, name)) for name in listdir(IMG_FOLDER) if path.isfile(path.join(IMG_FOLDER, name))]

print("All images")

File: clear_images.js
const fs = require('fs');
const path = require('path');
const Confirm = require('prompt-confirm');

const IMG_FOLDER = 'imgs';

const deleteAllImages = () => {
	fs.readdir(IMG_FOLDER, (err, files) => {
		if (err){
			throw err;
		}
	
		for (const file of files) {
			fs.unlink(path.join(IMG_FOLDER, file), err => {
				if (err){
					throw err;
				}
			});
		}
	});
}

console.log('Clearing all saved images')
new Confirm('Are you sure?')
	.run()
	.then(function(answer) {
		if (answer){
			deleteAllImages()
		}
	});
